thewalrus/tests/test_csamples.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2019 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""Tests for the classical sampling functions"""
+
+import numpy as np
+from thewalrus.csamples import (
+    rescale_adjacency_matrix_thermal,
+    generate_thermal_samples,
+)
+
+rel_tol = 10
+
+def generate_positive_definite_matrix(n):
```

Comment:
What about just hard-coding a 10-dimensional positive definite matrix?

Reply:
This will remove the randomness in the tests.

Reply:
The test is deterministic even though the inputs are not.

Reply:
Is there an advantage of using a random positive definite matrix?

Reply:
yes. The sampling algorithms that  work in polynomial time require PSD matrices.